src/components/UserBubble/UserBubble.vue

```diff
@@ -94,7 +94,7 @@ This component has the following slot:
 				class="user-bubble__avatar" />
 
 			<!-- Title -->
-			<h6 class="user-bubble__title">
+			<h6 class="user-bubble__title" role="presentation">
```

Comment:
Do we still need the h6 then? I am not sure that make sense if we strip of their default role

Reply:
we don't really, could use a div instead and need to check that the styling stays the same

Reply:
span then if it's just text